crypt.go

```diff
@@ -323,6 +454,100 @@ type ActivateVolumeOptions struct {
 	KeyringPrefix string
 }
 
+type activateVolumeWithKeyDataError struct {
```

Comment:
should this be exported, similar errors for TPMs are?

Reply:
I was going to leave this one unexported for now and maybe export it later if we need a way to introspect the errors, but I don't think it will be useful at the moment.